java/Cherepnin_Oleh/src/com/zoolatech/lecture3/tasks/_1/Orchestra.java

```diff
@@ -0,0 +1,28 @@
+package com.zoolatech.lecture3.tasks._1;
+
+import java.util.Set;
+
+public class Orchestra {
+    private Set<Musician> musicians;
+
+    public Orchestra(Set<Musician> musicians) {
```

Comment:
probably, it might be better to create a set instance inside of this class instead of passing it in the constructor to not expose its state